PC/pyconfig.h

```diff
@@ -507,7 +510,9 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 /* #undef HAVE_GETPGRP */
 
 /* Define if you have getpid.  */
-/* #undef HAVE_GETPID */
+#ifndef MS_WINDOWS_UWP
+#define HAVE_GETPID
+#endif
```

Comment:
I think I dislike this conditional define more than I dislike saying it's there when it's "not". Let's just leave it defined, and use the new variable to decide whether to actually use it or not (basically, minimise the use of the new variable, and keep it as close as possible to where it's working around a limitation)